calico_node/startup/startup_test.go

```diff
@@ -118,6 +119,13 @@ var _ = Describe("FV tests against a real etcd", func() {
 					if pool.Spec.IPIP != nil {
 						Expect(pool.Spec.IPIP.Enabled).To(BeFalse())
 					}
+
+					if expectedIPV6NATOutgoing {
```

Comment:
I think it would be better to replace this if statement and the nested checks with
`Expect(pool.Spec.NATOutgoing).To(Equal(expectedIPv4NATOutgoing))`